charts/pega/README.md

```diff
@@ -265,7 +265,7 @@ Provider  | Kubernetes Secrets | Cloud SSL management service
 ---       | ---                | ---
 AKS       | Supported          | None
 EKS       | Not supported      | Manage certificate using Amazon Certification Manager and use ssl_annotation - see example for details.
-PKS       | Supported          | None
+TKGI       | Supported          | None
```

Comment:
According to the earlier section, the provider *value* is still `pks`, while the *target* is TKGI... Maybe we need to unify this a bit.

Reply:
the text is updated; I don't know if anyone has updated with provider text, so I can make further changes if that is required.